src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.props

```diff
@@ -74,6 +74,23 @@ Copyright (c) .NET Foundation. All rights reserved.
     <None Remove="**\*.razor" />
   </ItemGroup>
 
+  <!-- Properties related to handling static web assets in the package  -->
+  <PropertyGroup>
+    
+    <!-- Used to indicate that a $(PackageId).props importing Microsoft.AspNetCore.StaticWebAssets.props file should be
+         automatically generated and included in the package under build\$(PackageId).props -->
+    <StaticWebAssetsEnableProjectBuildPropsFileGeneration>true</StaticWebAssetsEnableProjectBuildPropsFileGeneration>
```

Comment:
You don't really need to be declare. Just need to make sure they don't evaluate to 'false' prior to generating these files.

Reply:
Yeah, but I find declaring them to be more explanatory. Is that ok?

Reply:
In particular because these are knobs for individual bits, it's less useful. What might be more useful is to add to the docs here: https://docs.microsoft.com/en-us/aspnet/core/razor-pages/sdk?view=aspnetcore-2.2#properties. It's much more useful to point to this when a user pops up than telling users to dig around the SDK.

Reply:
Fair enojgh